crates/deltalake-core/src/delta_datafusion/logical.rs

```diff
@@ -1,5 +1,7 @@
 //! Logical Operations for DataFusion
 
+use std::collections::HashSet;
```

Comment:
Should we use hashbrown here instead? 

I read somewhere it's the default in a newer version in the std library but not sure.

Reply:
Yah as you called out they are changing the underlying implementation to be hashbrown